bxin/ptg2hex/hex_diagnostics.ipynb

```diff
@@ -22,6 +22,19 @@
     "#hexIdx = 2 #M2 hexapod"
```

Comment:
renamed to hexa. I am not using hex_1 because this notebook should work for both hexapod. Using hex_1 gives the impression this is only for the Camera Hexapod.